client-v2/src/bundles/frontsUIBundle.ts

```diff
@@ -346,7 +383,7 @@ const reducer = (state: State = defaultState, action: Action): State => {
     }
     case REMOVE_SUPPORTING_ARTICLE_FRAGMENT:
     case REMOVE_GROUP_ARTICLE_FRAGMENT:
-    case REMOVE_CLIPBOARD_ARTICLE_FRAGMENT: {
+    case 'REMOVE_CLIPBOARD_ARTICLE_FRAGMENT': {
       const articleFragmentId = action.payload.articleFragmentId;
       const selectedFrontId = Object.keys(state.selectedArticleFragments).find(
```

Comment:
This is a string to remove a circular dependency -- handling this and keeping the constant would mean a bigger refactor, but we should definitely deal with this down the line in a more referential way.